### ISSUE TITLE
Root and Attachment Folder Icons

### ISSUE THREAD

**Issue by qazzym** [notaapp/talk#295]
In my sidebar I have root, attachments and a few other folders. I have noticed a bug whereby the icons next to root and attachments occasionally display as folder icons instead of their usual custom icons. 

I am not able to reproduce this behaviour reliably so I will keep experimenting but just wanted to bring it to your attention.

**Comment by astoilkov** [comment]
Thanks. Can you share your configuration?

**Comment by qazzym** [comment]
I think I've been able to replicate it. It happens when I open Nota and it already has a file I'm editing outside of the workspace loaded in. In that case the icons look like the following: 

<img width="233" alt="Screenshot 2022-08-20 at 6 01 42 pm" src="https://user-images.githubusercontent.com/45447185/185758530-d181070d-5b17-4bdd-894f-366541aa32c8.png">

Otherwise the icons are: 

<img width="228" alt="Screenshot 2022-08-20 at 6 02 59 pm" src="https://user-images.githubusercontent.com/45447185/185758542-fe6cd2c9-fb6f-4fae-a5b0-d0d508e93128.png">

I have also noticed that when I am editing a file outside the workspace that file can become 'sticky' in that even when I close its tab before quitting Nota, it will occasionally load back in when I restart the app.
